src/components/Comments/RemoveComment.jsx

```diff
@@ -4,30 +4,36 @@ import Box from "@mui/material/Box";
 import { removeComment } from "../api";
 import { useState } from "react";
 
-export default function RemoveComment({
+export default function DeleteSelectedComment({
   commentId,
   setRenderToggle,
   setNumberofComments,
+  setOpen
 }) {
   const [loading, setLoading] = useState(false);
 
-  const handleClick = () => {
+  const handleCommentDeletion = () => {
     setLoading(true);
     removeComment(commentId).then(() => {
+        setOpen((open) => {
+            return true
+        })
       setNumberofComments((currNumber) => {
         return currNumber - 1;
       });
       setRenderToggle((toggle) => {
         return !toggle;
       });
       setLoading(false);
-    });
+    }).catch((err) => {
+      alert('Comment deletion failed')
+    })
```

Comment:
Good feedback to give, but ideally we'd avoid using alerts since they're notorious for being annoying and not being great for the user experience